README.md

```diff
@@ -37,6 +37,9 @@ Example: Limit on accepted emails per hour from a given user, by defining a thre
 
 ```ruby
 Prop.configure(:mails_per_hour, threshold: 100, interval: 1.hour, description: "Mail rate limit exceeded")
+
+# Disable by setting threshold to 0
```

Comment:
We're not disabling when setting to 0.. it's the opposite.. we're blocking all requests by setting to 0 :)